content/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account.md

```diff
@@ -24,7 +24,7 @@ After adding a new SSH key to your {% data variables.product.product_name %} acc
   If your SSH key file has a different name than the example code, modify the filename to match your current setup. When copying your key, don't add any newlines or whitespace.
 
   ```shell
-  $ pbcopy &lt; ~/.ssh/id_rsa.pub
+  $ pbcopy &lt; ~/.ssh/id_ed25519.pub
   # Copies the contents of the id_rsa.pub file to your clipboard
```

Comment:
```suggestion
  # Copies the contents of the id_ed25519.pub file to your clipboard
```

Could you update this line to match your other change? 😊

Reply:
Hi @brockneedscoffee - there seem to be several other problems with this topic, so I'm going to commit the reviewer's suggestion and merge your PR to unblock fixes to other areas of the topic.